include/vcpkg/base/messages.h

```diff
@@ -758,6 +793,15 @@ namespace vcpkg
                     "",
                     "A downloadable copy of this tool is available and can be used by unsetting {env_var}.");
     DECLARE_MESSAGE(DownloadedSources, (msg::spec), "", "Downloaded sources for {spec}");
+    DECLARE_MESSAGE(DownloadingPortableToolVersionX,
+                    (msg::tool_name, msg::version),
+                    "",
+                    "A suitable version of {tool_name} was not found (required v{version}) Downloading "
+                    "portable {tool_name} v{version}...");
+    DECLARE_MESSAGE(DownloadingTool,
+                    (msg::tool_name, msg::url, msg::path),
+                    "",
+                    "Downloading {tool_name}...\n{url}->{path}");
```

Comment:
```suggestion
                    "Downloading {tool_name}...\n{url} -> {path}");
```

Spaces are needed because leaving them out will result in a wrong URL.